python/CSXCAD/CSRectGrid.pyx

```diff
@@ -167,11 +163,10 @@ cdef class CSRectGrid:
         """
         self.thisptr.clear()
 
-    def SetDeltaUnit(self, unit):
-        """ SetDeltaUnit(unit)
-
-        Set the drawing unit for all mesh lines. Default is 1 (m)
-        """
+    def SetDeltaUnit(self, unit:float=1):
```

Comment:
I don't like this default? If the user uses SetDeltaUnit  it should require a value to be set? Otherwise it might be unclear what happens?

Reply:
Agree. I somehow confused by the "default is 1 (m)" from the docstring, but that is something different. Will change it.